fv3config/fv3run/_native.py

```diff
@@ -19,7 +21,28 @@
 logger = logging.getLogger("fv3run")
 
 
-def run_native(config_dict_or_location, outdir, runfile=None):
+def call_via_subprocess(func):
+    this_module = func.__module__
+
+    def main(argv):
+        import json
+
+        args, kwargs = json.loads(argv[1])
+        func(*args, **kwargs)
+
+    def command(*args, **kwargs) -> str:
```

Comment:
can use `@functools.wraps` to give this function the same call signature as `run_native`. You may want to figure out how to just modify the call signature without modifying the name or docstring, and see if there are any side-effects I don't know about.

Reply:
I added the decorator. However, after reading the [docs](https://docs.python.org/2/library/functools.html#functools.update_wrapper), I don't think `functools.wraps` does any sort of argument checking. Do you think that feature is important to include, since atm this is not a public API, and we can just assume that run_k8s and run_docker know what they are doing?